Jenkinsfile.prod

```diff
@@ -0,0 +1,23 @@
+elifePipeline {
+    def commit
+    stage 'Checkout approved', {
+        checkout scm
+        commit = elifeGitRevision()
+        // temporary hardcoding to approved
+        commit = 'approved'
+    }
+
+    stage 'Deploy to demo', {
+        lock ('data-pipeline--demo') {
+            elifeDeploySlackNotification 'data-pipeline', 'demo'
+            //elifeGitMoveToBranch commit, 'master'
```

Comment:
what does `elifeGitMoveToBranch` do? a merge into that branch? Will we activate that in the future?

Reply:
Merge the commit into `master`. I think we can enable it now, since it keeps the branch up to date with this pseudo-production environment.